src/CNCULT2.CPP

```diff
@@ -377,15 +377,21 @@ RC TOPRAT::tp_SetCheckTimeSteps()	// initialize timesteps
 	tp_subhrDur = 1.f / tp_nSubSteps;			// subhr duration
 
 	// substep ticks = even shorter steps for e.g. HPWH simulation
-	tp_nSubhrTicks = max(1, 60 / tp_nSubSteps);
+	if (!IsSet(tp_nSubhrTicks)) { // Not set
+		tp_nSubhrTicks = max(1, 60 / tp_nSubSteps); // Default
+	}
+	else if (tp_nSubhrTicks != max(1, 60 / tp_nSubSteps)) {
+		warn("This input is currenlty used only for experimental purposes");
```

Comment:
@galanCA have you triggered this warning? How does it appear to a user? Is there enough context to understand the message?